docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -100,6 +100,16 @@ pre[class*="language-"] {
     cursor: help;
 }
 
+.language-abnf .token {
+    display: inline;
+    padding: 0;
+    margin: 0;
+}
+
+pre {
+    counter-reset: lineNumber;
+}
+
```

Comment:
I'm not sure if the changes in css are necessary now, since the formatting issues have been fixed by https://github.com/eslint/eslint/commit/e6cb05aa35bafb9e88f161ad1fa6b01942a7c13c and https://github.com/eslint/eslint/commit/293573eb530d161d2a5b01efd9d3de49dadea022 in the meantime:

https://eslint.org/docs/head/developer-guide/package-json-conventions